app/cloud.go

```diff
@@ -62,57 +121,146 @@ func (a *App) NotifySystemAdminsToUpgrade(c *request.Context, currentUserTeamID
 		return appErr
 	}
 
+	now := model.GetMillis()
+
+	data, err := a.Srv().Store.NotifyAdmin().Get(trial)
+	if err != nil {
+		return model.NewAppError("SendNotifyAdminPosts", "app.notify_admin.send_notification_post.app_error", nil, err.Error(), http.StatusInternalServerError)
+	}
+
+	data = filterNotificationData(data, func(nad *model.NotifyAdminData) bool { return string(nad.RequiredPlan) != currentSKU })
```

Comment:
Should we also check if they are on trial? If they already on Enterprise trial, we should notify on that

Reply:
Being on trial means the current SKU would be `enteprise`. I think it makes sense not to send notifications to trial a plan they're already trialing

Reply:
Correct, I meant we shouldn't notify if they are already on trial. Good point that if they are on trial they are already in Enterprise 👍